process/incubation.md

```diff
@@ -167,6 +181,20 @@ The SSWG will publicly share a list of vulnerabilities and fixes on [Swift forum
 
 Project authors are also encouraged to make use of their source control system security features (for example: Github's "Security Advisories" and Gitlab's "Confidential Issues") to manage the vulnerabilities and inform their users.
 
+## Technical Best Practices
+
+* Prefer native Swift over C wrapping, where appropriate
+* Concurrency / IO
+  * Packages should be non-blocking (w/ async API) unless not possible (blocking C libs, etc)
+  * There should be as little (preferably no) wrapping of NIO as possible. Exposing NIO types directly will go a long way for making packages compatible.
+  * Blocking code should be wrapped in NonBlockingIOPool (like Vapor's SQLite package)  
+* Uses force unwraps and force tries only as preconditions, ie. conditions that the programmer regards as impossible or programmer error. All force tries/unwraps should come with a comment stating the reasons
```

Comment:
Use rather than Uses?  Similarly Do rather than Does below?  Feels like the subject of this section changes as you read through it to me although I wouldn't claim expertise at written text.